test/integration/multiple_deposit.test.ts

```diff
@@ -125,169 +143,793 @@ contract<ERC20RootVault, DeployOptions, CustomContext>(
+
+                    if (
+                        BigNumber.from(delay).gt(
+                            (
+                                await this.erc20RootVaultGovernance.delayedProtocolParams()
+                            ).managementFeeChargeDelay
+                        )
+                    ) {
+                        let vaultGovernanceManagementFee = (
+                            await this.erc20RootVaultGovernance.delayedStrategyParams(
+                                this.erc20RootVaultNft
+                            )
+                        ).managementFee;
+                        let realPerformanceFee = await this.subject.balanceOf(
+                            strategyPerformanceTreasury
+                        );
+                        let realManagementFee = await this.subject.balanceOf(
+                            strategyTreasury
+                        );
+
+                        // calculate expected management fee after second deposit
+                        let expectedManagementFeeFirst =
+                            vaultGovernanceManagementFee
+                                .mul(delay)
+                                .mul(lpTokenAmountAfterFirstDeposit)
+                                .div(
+                                    BigNumber.from(10)
+                                        .pow(9)
+                                        .mul(24)
+                                        .mul(3600)
+                                        .mul(365)
+                                );
+
+                        // calculate expected management fee after first withdraw
+                        let expectedManagementFeeSecond =
+                            vaultGovernanceManagementFee
+                                .mul(delay)
+                                .mul(baseSupply)
+                                .div(
+                                    BigNumber.from(10)
+                                        .pow(9)
+                                        .mul(24)
+                                        .mul(3600)
+                                        .mul(365)
+                                );
+
+                        // expected management fee = first fee + second fee
+                        let expectedManagementFee =
+                            expectedManagementFeeFirst.add(
+                                expectedManagementFeeSecond
+                            );
+
+                        let managementFeeAbsDifference = expectedManagementFee
+                            .sub(realManagementFee)
+                            .abs();
+
+                        /* 
+                            MANAGEMENT FEES
+                            dif <= 0.01 * 1% * fee
+                            dif * 100 * 100 <= fee
+                            dif * (10 ** 4) - fee <= 0
+                        */
+                        expect(
+                            managementFeeAbsDifference
```

Comment:
The abs difference should be small enough. Can we convert it to floats?

Reply:
IMO no need to use floats here because it there is a BigNumber in calculations => we must use x * 10**6 / 10**6

Right now we do not use rounded values, which IMO is better